releasenotes/notes/upgrade-jmxfetch-0-47-9-46ffbe691f9e5eca.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Upgraded JMXFetch to ``0.47.9`` which has fixes to improve
+    efficiency when fetching beans, a fix not leak threads in
```

Comment:
```suggestion
    efficiency when fetching beans, a fix for leak threads in
```

Reply:
I'm not sure if replacing "not" with "for" is correct, but the wording reads a bit odd to me. Let me know if I'm mistaken and it's correct.

Reply:
Updated this, it did read a bit weird 👍